packages/styles/scss/components/masthead/_masthead-l1.scss

```diff
@@ -66,6 +68,7 @@ $search-transition-timing: 95ms;
       color: $text-01;
       position: relative;
       text-decoration: none;
+      //white-space: nowrap;
```

Comment:
minor thing, if this isn't being used we should remove this.

Reply:
Thanks! Will update!